### ISSUE TITLE
Self-supervised pre-training - how to load the dataset?

### ISSUE THREAD

**Issue by zhanglaoban-kk** [junchen14/LoMaR#4]
Hi！
I want to use your project for self-supervised pre-training, but when loading the data, it prompts that the specified path cannot be found, my data is unlabeled picture data all in one folder, and the training set and the test set are not divided? Is it necessary to divide when loading, but if it is divided, isn't there label data?
Hope to hear from you！
